### PR TITLE
[libc] fix readability-identifier-naming.ConstexprFunctionCase

### PR DESCRIPTION
**Pull request by nickdesaulniers**
Codify that we use lower_case for
readability-identifier-naming.ConstexprFunctionCase and then fix the 11
violations (rather than codify UPPER_CASE and have to fix the 170 violations).

### CODE REVIEW

File: libc/src/__support/FPUtil/FPBits.h [hunk 1]
```diff
@@ -239,23 +239,23 @@ template <FPType fp_type> struct FPStorage : public FPLayout<fp_type> {
...
+    LIBC_INLINE static constexpr auto zero() { return Exponent(0); }
+    LIBC_INLINE static constexpr auto max() { return Exponent(EXP_BIAS); }
+    LIBC_INLINE static constexpr auto inf() { return Exponent(EXP_BIAS + 1); }
   };
 
   // An opaque type to store a floating point biased exponent.
   // We define special values but it is valid to create arbitrary values as long
-  // as they are in the range [BITS_ALL_ZEROES, BITS_ALL_ONES].
-  // Values greater than BITS_ALL_ONES are truncated.
+  // as they are in the range [BITS_ALL_zeroES, bits_all_ones].
```
Comment (nickdesaulniers) [inline libc/src/__support/FPUtil/FPBits.h:257]: BITS_ALL_ZEROS doesn't appear anywhere in sources. Should this just have been `0`?

Comment (nickdesaulniers) [reply libc/src/__support/FPUtil/FPBits.h:257]: ah, `zero`

Comment (nickdesaulniers) [reply libc/src/__support/FPUtil/FPBits.h]: cf7928261fb4

**Comment by nickdesaulniers** [comment]
Some other arch specific files need to be fixed up here, too.

**Review by michaelrj-google — APPROVED** [review]
LGTM

File: libc/src/string/memory_utils/generic/byte_per_byte.h [hunk 2]
```diff
@@ -56,8 +56,8 @@ inline_bcmp_byte_per_byte(CPtr p1, CPtr p2, size_t count, size_t offset = 0) {
   LIBC_LOOP_NOUNROLL
   for (; offset < count; ++offset)
     if (p1[offset] != p2[offset])
-      return BcmpReturnType::NONZERO();
-  return BcmpReturnType::ZERO();
+      return BcmpReturnType::zero();
```
Comment (nickdesaulniers) [inline libc/src/string/memory_utils/generic/byte_per_byte.h:59]: oops!
https://github.com/llvm/llvm-project/pull/83356

**Pull request merged**